src/spikeinterface/preprocessing/whiten.py

```diff
@@ -185,6 +189,7 @@ def compute_whitening_matrix(
 
     """
     random_data = get_random_data_chunks(recording, concatenated=True, return_scaled=False, **random_chunk_kwargs)
```

Comment:
Could someone remind me why we want to `return_scaled=False` here?

Reply:
because scaling only happens at the `BaseRecording` step. The whitening is happening at the `BaseSegment` step, which returns unscaled traces

Reply:
Cool. Thanks.